tests/test_visitors/test_tokenize/test_conditions/test_implicit_elif.py

```diff
@@ -46,6 +46,30 @@
         ...
 """
 
+# False positives
```

Comment:
Can you please provide a link to the original issue here?

Reply:
Ok!